pkg/cmd/extension/command.go

```diff
@@ -39,6 +45,148 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 	}
 
 	extCmd.AddCommand(
+		func() *cobra.Command {
+			query := search.Query{
+				Kind: search.KindRepositories,
+			}
+			qualifiers := search.Qualifiers{
+				Topic: []string{"gh-extension"},
+			}
+			var order string
+			var sort string
+			var webMode bool
+			var exporter cmdutil.Exporter
+
+			cmd := &cobra.Command{
+				Use:   "search [<query>]",
+				Short: "Search available gh extensions",
+				Long: heredoc.Doc(`
+				Search for gh extensions.
+
+				This command behaves similarly to 'gh search repos' but does not
+				support as many search qualifiers. For a finer grained search of
+				extensions, try using:
+
+				gh search repos --topic "gh-extension"
```

Comment:
To render as a code block, this could be indented by a tab deeper than other text